console/cli_context.go

```diff
@@ -51,3 +192,35 @@ func (r *CliContext) OptionInt64(key string) int64 {
 func (r *CliContext) OptionInt64Slice(key string) []int64 {
 	return r.instance.Int64Slice(key)
 }
+
+func (r *CliContext) Secret(question string, option ...console.SecretOption) (string, error) {
+	var answer string
+	if len(option) > 0 {
+		answer = option[0].Default
+	}
+
+	input := huh.NewInput().Title(question)
+
+	if len(option) > 0 {
+		input.CharLimit(option[0].Limit).Description(option[0].Description).Placeholder(option[0].Placeholder).Password(true)
+		if option[0].Validate != nil {
+			input.Validate(option[0].Validate)
+		}
+	}
+
+	err := input.Value(&answer).Run()
+	if err != nil {
+		return "", err
+	}
+
+	return answer, nil
+}
+
+func (r *CliContext) Spinner(message string, option ...console.SpinnerOption) error {
```

Comment:
How to use this function?

Reply:
Can we implement it like below?
<img width="684" alt="image" src="https://github.com/goravel/framework/assets/24771476/c3b50963-ae2a-4be0-a7fb-99aeac99fb0d">

Reply:
Actually spinner is different from progressBar. I will try to implement it similar to image.